pages/index.js

```diff
@@ -27,13 +27,10 @@ export async function getServerSideProps() {
     `${process.env.API_URL}/organizaciones-politicas`,
   );
   const politicalParties = await responsePoliticalParties.json();
-  const responseCandidates = await fetch(`${process.env.API_URL}/congresistas`);
```

Comment:
We could prefetch data and set it on the react-query cache with the configuration for nextjs https://react-query.tanstack.com/docs/guides/ssr#integrating-with-nextjs